.vscode/controls.code-snippets

```diff
@@ -42,7 +42,7 @@
 			"    objective: <1-3 sentence description of the control objective>",
 			"    control_family: <Control Family>",
 			"    nist_csf: <NIST CSF control ID>",
```

Comment:
I don't think this field exists anymore, should be part of `guideline-mappings` now